javascript/ql/lib/semmle/javascript/frameworks/SQL.qll

```diff
@@ -420,6 +442,11 @@ private module MsSql {
       mssql().getMember("query").getAUse() = DataFlow::valueNode(astNode.getTag())
     }
 
+    override DataFlow::Node getAResult() {
+      PromiseFlow::loadStep(this.(DataFlow::SourceNode).getALocalUse(), result,
```

Comment:
A `DataFlow::valueNode` where the `astNode` is of type `TaggedTemplateExpr` is always a `DataFlow::SourceNode` (see the `DefaultRange` class in `Sources.qll`).  
So you should be able to add `DataFlow::SourceNode` on top of the other classes in the extends clause. 

(Just to get rid of the inline cast to `DataFlow::SourceNode`).

Reply:
done!